FreeRTOS/FreeRTOS-Plus-TCP
# Update debug log levels and fix STM32F port build when `ipconfigSUPPORT_NETWORK_DOWN_EVENT` is enabled

Description
-----------
This PR updates the logging APIs used for some of the debug logs to `FreeRTOS_debug_printf` instead of `FreeRTOS_printf` to reduce the verbosity of logs when `ipconfigHAS_DEBUG_PRINTF` is disabled,

Test Steps
-----------
Run demo on STM32F4.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
